test/test_mps.py

```diff
@@ -162,6 +162,13 @@ def mps_ops_grad_modifier(ops):
 
     XPASSLIST_GRAD = {
         'nn.functional.pairwise_distance': [torch.float16],
+        # failed assertion `destination datatype must be fp32'
```

Comment:
what's the problem here?

Reply:
The error is : 
```
test_mps.py::TestConsistencyCPU::test_output_grad_match_nn_functional_conv1d_cpu_float16 /AppleInternal/Library/BuildRoots/c651a45f-806e-11ed-a221-7ef33c48bc85/Library/Caches/com.apple.xbs/Sources/MetalPerformanceShaders/MPSNDArray/Kernels/MPSNDArrayConvolutionA14.mm:3976: failed assertion `destination datatype must be fp32'
```
mps does not support fp16 conv.